pkg/flag/db_flags.go

```diff
@@ -50,17 +50,17 @@ var (
 		ConfigName: "db.no-progress",
 		Usage:      "suppress progress bar",
 	}
-	DBRepositoryFlag = Flag[string]{
+	DBRepositoryFlag = Flag[[]string]{
 		Name:       "db-repository",
 		ConfigName: "db.repository",
-		Default:    db.DefaultRepository,
-		Usage:      "OCI repository to retrieve trivy-db from",
+		Default:    []string{db.DefaultGHCRRepository, db.DefaultECRRepository},
```

Comment:
I think we can say that we keep order when trying to download the db (as for `--image-src`).